app/src/main/java/com/duckduckgo/app/cta/ui/CtaViewModel.kt

```diff
@@ -292,12 +401,14 @@ class CtaViewModel @Inject constructor(
                 userStageStore.stageCompleted(AppStage.DAX_ONBOARDING)
                 false
             }
+
             else -> true
         }
     }
 
     private suspend fun canShowPrivacyProCta(): Boolean {
-        return daxOnboardingActive() && !hideTips() && !daxDialogPrivacyProShown()
+        return daxOnboardingActive() && !hideTips() && !daxDialogPrivacyProShown() &&
+            extendedOnboardingFeatureToggles.testPrivacyProOnboardingCopyNov24().isEnabled()
```

Comment:
I'm not sure this works as you expect. Testing cases says the PPro dialog should be shown to returning users but here you are checking if the sub-feature is enabled which will always be false for returning users because of the targets.